charts/rancher-k3s-upgrader/0.4.0/templates/validate-psp-install.yaml

```diff
@@ -0,0 +1,7 @@
+#{{- if gt (len (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "")) 0 -}}
```

Comment:
Do you mean to check if there is any ClusterRole or if the particular one named `system-upgrade-controller-psp` is in the cluster?

Reply:
This is the standard validation file that's now being used across all charts, from what I understand its required to ensure that `helm template` will still work with these conditions added

Reply:
OK. Though what it checks seems useless to me:  check if there is any `ClusterRole.rbac.authorization.k8s.io/v1` object in the cluster, it does not hurt at least 😆